### ISSUE TITLE
Add automated testing

### ISSUE THREAD

**Issue by robintw** [pmlrsg/PySpectra#5]
I think we should add some tests, and ideally get them running on a continuous integration system.

We'd need to include (at least one) file for each format that we're reading, and check we read it correctly. I'll start doing this when I add some more formats - but it'd be great if you could find a `.sig` file we can use as a test.

My proposal is to have a `test` directory with the tests in it, and maybe the files in `test/inputs` or something?

As for automation/continuous integration: hopefully something like Travis CI would work, but if not then I have a server running Jenkins to do Py6S testing (which is a bit too complex for the free Travis plan) which we could use.

**Comment by danclewley** [comment]
Yes, tests are a good idea. I'll get an example for the `.sig` format added.

**Comment by robintw** [comment]
I've added a test for the USGS format. I've ended up hard-coding in the array values that it should be reading, as I can't think of a better way to test it - even though it makes the file a bit messy.

Also, I've defined a useful constant at the top called `TEST_INPUTS_DIRECTORY` which should make the tests work wherever they're run from. They all work fine at the moment when called with `nosetests` from the root directory of the repo.
